CrewChiefV4/Events/MandatoryPitStops.cs

```diff
@@ -143,6 +147,34 @@ public override void clearState()
             enableWindowWarnings = true;
         }
 
+        public override bool isMessageStillValid(String eventSubType, GameStateData currentGameState, Dictionary<String, Object> validationData)
+        {
+            if (base.isMessageStillValid(eventSubType, currentGameState, validationData))
+            {
+/*              // When a new penalty is given we queue a 'three laps left to serve' delayed message.
```

Comment:
why is this commented out?

Reply:
Because I do not have sounds yet and I was planning to use eventSubType to decide what is being validated.  All in all, maybe we should wait till there are sounds.  But, I wanted to get feedback as well.